docs/src/docs/asciidoc/index.adoc

```diff
@@ -521,8 +521,8 @@ detailed information on configuration options for Hazelcast.
 
 Sessions will be stored in a distributed `Map` in Hazelcast using a <<api-mapsessionrepository,MapSessionRepository>>.
 The `Map` interface methods will be used to `get()` and `put()` Sessions.
-The expiration of a session in the `Map` is handled by Hazelcast's native support for configuring a map's `max-idle-seconds` setting.
-Entries (sessions) that have been idle longer than the max idle setting will be automatically removed from the `Map`.
+The expiration of a session in the `Map` is handled by Hazelcast's support for setting the time to live on an entry when it is `put()` into the `Map`.
```

Comment:
As [discussed in Gitter](https://gitter.im/spring-projects/spring-session?at=56b07d126b6468374a0af02c), this is not an issue as I thought it was. Things should work as expected, but it would be better if we add a test specifically for this, like:
* Save a session
* Wait a some time less than the expiration time
* Update the session as SessionRepositoryFilter does
* Wait until past the original time it should have expired
* Ensure it has not expired because it was updated.